Components/cart/cart.html

```diff
@@ -0,0 +1,118 @@
+      integrity="sha512-BnbUDfEUfV0Slx6TunuB042k9tuKe3xrD6q4mg5Ed72LTgzDIcLPxg6yI2gcMFRyomt+yJJxE+zJwNmxki6/RA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <nav class="navigation flex-row space-between align-center">
+      <div class="left-navigation flex-row align-center">
+        <i class="fas fa-bars hamburger" id="hamburger"></i>
+        <img
+          src="../../assets/20220206_152352.png"
+          class="big-logo"
+          alt="logo"
+        />
+        <img src="../../assets/20220203_122320.png" class="logo" alt="logo" />
+      </div>
+
+      <div class="nav-links nav-links-not-visible">
+        <a href="../../index.html">Home</a>
+        <a href="../product/product.html">Browse</a>
+      </div>
+
+      <div class="right-navigation navigation-icons flex-row align-center">
+        <a href="../../authentication.html"
+          ><button class="btn btn-primary">Login</button></a
+        >
+
+        <a href="../wishlist/wishlist.html" class="badge-on-icon">
+          <i class="far fa-bookmark"></i
+          ><span class="badge badge-primary icon-badge">4</span>
+        </a>
+
+        <a href="#" class="badge-on-icon">
+          <i class="fas fa-shopping-cart"></i>
+          <span class="badge badge-primary icon-badge">4</span>
+        </a>
+      </div>
+    </nav>
+
+    <main>
+      <section class="product-list-title">
+        <div class="cart-container">
+          <div class="cart-info">
+            <p class="text-lg total">Total</p>
+            <div class="cart-total-price">
+              <span class="selling-price black text-md">₹1,600</span
+              ><span class="cost-price">₹3,200</span><span>50% off</span>
+            </div>
+            <button class="btn btn-secondary">Checkout</button>
+          </div>
+
+          <div class="cart-items">
+            <div class="card card-shadow vertical-card">
+              <div class="card-header">
+                <span class="badge badge-primary">50% off</span>
+                <img
+                  src="https://cdn1.epicgames.com/epic/offer/RDR2PC1227_Epic Games_860x1148-860x1148-b4c2210ee0c3c3b843a8de399bfe7f5c.jpg?h=854&resize=1&w=640"
+                  alt="Avatar"
+                  class="card-image"
+                />
+              </div>
+              <div class="card-content">
+                <div class="card-title">Red Dead Redemption 2</div>
+                <div class="card-subtext">
+                  <span class="cost-price">₹3,200</span
+                  ><span class="selling-price">₹1,600</span>
+                </div>
+              </div>
+              <div class="item-quantity-selector">
+                <label class="item-qty-label" for="item-qty"
+                  >Quantity:<input
+                    type="number"
+                    name="item-qty"
+                    class="item-qty"
+                    id="item-qty"
```

Comment:
For input, add starting `value=1` and also set `min=1`. It is also taking negative values right now.

Reply:
Done